Blimp/Blimp.cpp

```diff
@@ -259,28 +248,22 @@ void Blimp::update_altitude()
     }
 }
 
-// vehicle specific waypoint info helpers
-bool Blimp::get_wp_distance_m(float &distance) const
+//Conversions are in 2D so that up remains up in world frame when the blimp is not exactly level.
+void Blimp::rotate_BF_to_NE(float &x, float &y)
 {
-    // see GCS_MAVLINK_Blimp::send_nav_controller_output()
-    distance = flightmode->wp_distance() * 0.01;
-    return true;
+    float ne_x = x*ahrs.cos_yaw() - y*ahrs.sin_yaw();
```

Comment:
we have a rotate_xy() function that you could use

Reply:
I actually just used the same method as Copter's "rotate_body_frame_to_NE()" function as having that function call seemed to make the code more readable, but if you'd still prefer I use this function then I could?